pkgs/development/compilers/blueprint/default.nix

```diff
@@ -29,18 +30,32 @@ stdenv.mkDerivation rec {
 
   buildInputs = [
     python3
-  ];
+    gtk4
+  ] ++ (with python3.pkgs; [
+    pygobject3
+    wrapPython
```

Comment:
`wrapPython` code is executed at build time so it should go to `nativeBuildInputs`.